dashboard/test/ui/features/teacher_tools/level_summary.feature

```diff
@@ -0,0 +1,29 @@
+@eyes
+Feature: Level summary
+
+Scenario: Free Response level 1
+  Given I am a teacher
+  And I create a new student section
+  Given I am on "http://studio.code.org/s/allthethings/lessons/27/levels/1/summary"
+  And I wait until element "#summary-container" is visible
+  And I wait to see ".uitest-sectionselect"
+  And I see no difference for "level summary"
+  And I close my eyes
+
+Scenario: Free Response level 2
```

Comment:
What's the difference between level 1, level 2, and level 3. Do we need all three?

Reply:
I picked these 3 because one has a `level.title` that gets rendered as an h1, one has no `level.title` but an h2 in the markdown, and one has no heading at all. i expect some heading differences with my next PR because of the `application.css` styles, and wanted to be sure to have baselines for that. the first `allthethings` also has more features used than the other 2, like the teacher instructions. other than that, I don't know what the best way to test this stuff is. it doesn't feel great to have such a wide variety of content in these levels with no way to find example levels that use each feature, I've just been asking Bryan or stumbling across levels by accident that have new things i've never seen before